examples/palsar2-scansar.json

```diff
@@ -0,0 +1,166 @@
+{
+  "type": "Collection",
+  "id": "palsar2-scansar",
+  "stac_version": "1.0.0",
+  "description": "The new PALSAR-2 ScanSAR products have been developed to conform with the CEOS-ARD format specifications for Normalised Radar Backscatter (CARD4L NRB). The NRB products have been subject to ortho-rectification and Radiometric Terrain Correction (RTC), and are provided in the Gamma-0 backscatter convention, which mitigates the variations from diverse observation geometries. The products include HH and HV polarisation backscatter, local incidence angle image and data mask.",
+  "links": [
+    {
+      "rel": "root",
+      "href": "/workspaces/palsar2-scansar/examples/palsar2-scansar.json",
+      "type": "application/json",
+      "title": "ALOS-2 PALSAR-2 ScanSAR"
+    },
+    {
+      "rel": "documentation",
+      "href": "https://www.eorc.jaxa.jp/ALOS/en/dataset/palsar2_l22_e.htm",
+      "type": "application/pdf",
+      "title": "The new PALSAR-2 ScanSAR products have been developed to conform with the CEOS-ARD format specifications for Normalised Radar Backscatter (CARD4L NRB). The NRB products have been subject to ortho-rectification and Radiometric Terrain Correction (RTC), and are provided in the Gamma-0 backscatter convention, which mitigates the variations from diverse observation geometries. The products include HH and HV polarisation backscatter, local incidence angle image and data mask.",
+      "description": "Also includes data usage information"
+    },
+    {
+      "rel": "license",
+      "href": "https://earth.jaxa.jp/policy/en.html",
+      "type": "JAXA Terms of Use of Research Data"
+    },
+    {
+      "rel": "self",
+      "href": "/workspaces/palsar2-scansar/examples/palsar2-scansar.json",
+      "type": "application/json"
+    }
+  ],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/projection/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/raster/v1.1.0/schema.json",
```

Comment:
Is the raster extension only included for the DataType in constants? Or is there hope that they'll fill in raster:bands?

Reply:
Should be fixed now.